### ISSUE TITLE
running one test file causes leftover lines

### ISSUE THREAD

**Issue by jeysal** [jest-community/jest-react-reporter#4]
![image](https://user-images.githubusercontent.com/16069751/68772897-81873d00-0622-11ea-809f-c06c37de4649.png)

Repro: https://github.com/yldio/yld.io/tree/38514750c72574747c96de405ab0ec174b75692d
```
yarn add -D jest-react-reporter@1.1.6
yarn test add-frontmatter --reporters=jest-react-reporter 
```

It also does not show test case details like the default reporter, not sure if it is supposed to do that.

**Comment by SimenB** [comment]
>It also does not show test case details like the default reporter, not sure if it is supposed to do that.

It's supposed to work the exact same, at least for a first iteration. Could you open up a separate issue for it? Or send a PR 🙂

**Comment by SimenB** [comment]
Haven't tried, but I'm wondering if it's related to https://github.com/vadimdemedes/ink/issues/162

**Comment by SimenB** [comment]
Haven't tried, but I'm wondering if it's related to https://github.com/vadimdemedes/ink/issues/162

**Comment by SimenB** [comment]
Testing now, it's something to do with `@testing-library/react`'s async act warning (just a `console.error`). Deleting that line from `node_modules` fixes the issue. Not sure why it's not properly intercepted...

**Comment by SimenB** [comment]
Figured out what's going on, although I have no idea how to fix it...

The issue is that, since we're only running a single test, Jest switches into `verbose` mode: https://github.com/facebook/jest/blob/18dfcbb41965a6c2682e1235b5f76dd071be24a1/packages/jest-core/src/runJest.ts#L231. Since we're `verbose`, we use the custom console instead of the buffered one: https://github.com/facebook/jest/blob/18dfcbb41965a6c2682e1235b5f76dd071be24a1/packages/jest-runner/src/runTest.ts#L135-L141. This console writes directly to stderr on log messages, rather than buffering them and relying on the reporter to print it: https://github.com/facebook/jest/blob/18dfcbb41965a6c2682e1235b5f76dd071be24a1/packages/jest-console/src/CustomConsole.ts#L46-L51

There is https://github.com/facebook/jest/issues/6441 which we should do, but it'll cause log entries to be printed twice... I think the only proper way to solve this is to land https://github.com/facebook/jest/pull/9001 and also stream individual console entries as they happen.

Thoughts? /cc @thymikee @rogeliog

**Comment by rogeliog** [comment]
Yes, I agree that we might need something like facebook/jest#9001 and facebook/jest#6441 would make sense for this.
